frontend/src/services/fetch/fetcherHelpers.ts

```diff
@@ -38,32 +37,6 @@ export function getDefaultHeaders(): HeadersDict {
   return headers;
 }
 
-/**
- * Send a request and handle the response
- * @param queryParamData: note that this is only used in error handling in order to help restore original page state
- */
-export async function sendRequest<ResponseType extends APIResponse>(
```

Comment:
since this function wasn't doing much, and what it was doing (reading a json response body) was making our implementation less flexible, I got rid of it and moved all this functionality either into the fetcher generator or endpoint specific wrappers